tensorflow/core/kernels/dml_reduce_ops.cc

```diff
@@ -261,7 +279,8 @@ class DmlReduceKernel : public DmlKernel {
         (reduce_helper.ndims() == 0 ||
          (reduce_helper.ndims() == 1 && !reduce_helper.reduce_first_axis()));
 
-    if (is_identity) {
```

Comment:
This case now will only be needed if output is 64 bit, since the identity case is handled by the kernel wrapper

Reply:
Why is it different for int64? If both the input and output are int64, and the operation is essentially identity, we should be able to just forward the input. The reason that we needed to convert from int32 to int64 here is that the result of identity would write to an unstrided int32 output that we would need to pad with leading 1's or 0's. But since the input is already a correctly padded int64 buffer, we should be able to reuse it as the output.